tools/workbench/src/main/java/org/eclipse/rdf4j/workbench/base/TransformationServlet.java

```diff
@@ -31,6 +32,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+@MultipartConfig
```

Comment:
@JervenBolleman @jeenbroekstra I think just adding this annotation to the base class for the multipart support servlets is not enough. It seems to me that this annotation is not marked as inherited. Since the test coverage in the workbench is rather poor, I think someone has to check this manually. And I have no idea how the workbench works and how to test it... Are you able to help, or pull in someone who can?

Reply:
I've fairly recently reinstalled my personal laptop and it seems I have completely messed up my docker installation. I'll try and sort it out ASAP and take a look. 

Fwiw I'm perfectly fine with just smoke-testing this: building the workbench, and just manually verifying that common functionality that is likely to use this (file uploads etc.) works as expected.

Reply:
Thx @jeenbroekstra, I also think smoke testing should be sufficient. This is just changing the technical foundation for multipart handling. So either all or nothing, works. 😉

Reply:
Move annotation to all concrete subckasses, as this annotation is not inherited.